### DISCUSSION TITLE
Create timelapse that isn't stacked

### DISCUSSION THREAD

**Discussion by Vertikar** [aaronwmorris/indi-allsky#415] [Q&A] [answered]
Hi All,

Just started using indi-allsky after using TJs for a while, and would like to also create a timelapse the same as TJs.
I've looked at the command that runs the timelapse [here](https://github.com/aaronwmorris/indi-allsky/blob/b677f03c804f2dfccb715339c916bb74472a161b/indi_allsky/timelapse.py#L41) but can't see the difference.
Can anyone assist?

TIA

**Comment by StarGeezerPhil** [comment, marked as answer]
Hi Vertikar,

It generates both (the stack is very newly added). Just click the keogram in your timelapse view to see the standard timelapse 👍

**Reply by Vertikar** [reply]
Ah, didn't even think to look there! Thanks
